ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -95,25 +95,35 @@ mkdir dev-scripts
 tar -xzvf dev-scripts.tar.gz -C /root/dev-scripts
 chown -R root:root dev-scripts
 
-NVME_DEVICE="/dev/nvme0n1"
-if [ -e "\$NVME_DEVICE" ];
+if [ ! -z "${NVME_DEVICE}" ] && [ -e "${NVME_DEVICE}" ];
 then
-  mkfs.xfs -f "\${NVME_DEVICE}"
+  mkfs.xfs -f "${NVME_DEVICE}"
   mkdir /opt/dev-scripts
-  mount "\${NVME_DEVICE}" /opt/dev-scripts
+  mount "${NVME_DEVICE}" /opt/dev-scripts
 fi
 
 cd dev-scripts
 
 cp /root/pull-secret /root/dev-scripts/pull_secret.json
 
-echo "export OPENSHIFT_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> /root/dev-scripts/config_root.sh
 echo "export ADDN_DNS=\$(awk '/nameserver/ { print \$2;exit; }' /etc/resolv.conf)" >> /root/dev-scripts/config_root.sh
 echo "export OPENSHIFT_CI=true" >> /root/dev-scripts/config_root.sh
 echo "export NUM_WORKERS=3" >> /root/dev-scripts/config_root.sh
 echo "export WORKER_MEMORY=16384" >> /root/dev-scripts/config_root.sh
 echo "export ENABLE_LOCAL_REGISTRY=true" >> /root/dev-scripts/config_root.sh
 
+if [[ "${ARCHITECTURE}" == "arm64" ]]; then
+  echo "export DOCKER_REGISTRY_IMAGE=quay.io/libpod/registry:2.8" >> /root/dev-scripts/config_root.sh
+  echo "export OPENSHIFT_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}" >> /root/dev-scripts/config_root.sh
+  echo "export IRONIC_LOCAL_IMAGE=\\\$(oc adm release info -a /root/dev-scripts/pull_secret.json \
+    \\\${OPENSHIFT_RELEASE_IMAGE} -o json | jq -r '.references.spec.tags[] | select(.name==\"ironic\").from.name')" \
+    >> /root/dev-scripts/config_root.sh
+  echo "export SUSHY_TOOLS_IMAGE=quay.io/multi-arch/sushy-tools:muiltarch" >> /root/dev-scripts/config_root.sh
+  echo "export VBMC_IMAGE=quay.io/multi-arch/vbmc:arm" >> /root/dev-scripts/config_root.sh
```

Comment:
We should find better locations for these.

Reply:
After an offline conversation, the `quay.io/multi-arch` registry is not considered a permanent solution, so these values should be updated with something more appropriate.

Reply:
I had discussed building these with some of the BM folks a while ago. The blocker was that they use [quay](https://quay.io/organization/metal3-io) for builds, and quay don't support arm builds ([yet](https://issues.redhat.com/browse/PROJQUAY-3059)). As a work around, I manually these to test. Happy to discuss more appropriate options.